opslevel/resource_opslevel_filter.go

```diff
@@ -64,6 +64,11 @@ var filterPredicateType = map[string]attr.Type{
 }
 
 func (fp filterPredicateModel) Validate() error {
+	// Key and Value are required fields, but may be unknown at validation time
+	// Creating multiple predicates with a 'for_each' is one example
+	if fp.Key.IsUnknown() || fp.Value.IsUnknown() {
```

Comment:
Seems like this might be true in other areas of the codebase.  Can we audit them and fix them in a follow up and if not trivial to fix like this cut a backlog ticket.

Reply:
[Ticket](https://github.com/OpsLevel/team-platform/issues/395) 👍